[BUG] Pose /lay and others not working on MC 1.17.1

**Describe the bug**
command `/lay` warnings

**To Reproduce**
Steps to reproduce the behavior:
1. Send `/lay`

**Plugin version:**
PosePlugin-4.0-Beta-1

**Server software**
Airplane 1.17.1 build 45

**Console log**
```
[Mon, 12. Jul 2021 12:59:33 UTC INFO] LinKs_QQ issued server command: /lay
[Mon, 12. Jul 2021 12:59:33 UTC WARN] java.lang.reflect.InvocationTargetException
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:78)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at ru.armagidon.poseplugin.api.poses.PoseBuilder.build(PoseBuilder.java:40)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at ru.armagidon.poseplugin.plugin.commands.PoseCommandGenerator.lambda$generatePoseCommands$4(PoseCommandGenerator.java:63)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at ru.armagidon.poseplugin.plugin.commands.SimpleCommand.lambda$new$1(SimpleCommand.java:39)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:821)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.server.network.PlayerConnection.handleCommand(PlayerConnection.java:2243)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2054)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2035)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:46)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:6)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$ensureRunningOnSameThread$1(PlayerConnectionUtils.java:56)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.server.TickTask.run(TickTask.java:18)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.util.thread.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:149)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.executeTask(IAsyncTaskHandlerReentrant.java:23)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:1407)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:190)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.util.thread.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:122)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.server.MinecraftServer.bf(MinecraftServer.java:1385)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.server.MinecraftServer.executeNext(MinecraftServer.java:1378)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.util.thread.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:132)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.server.MinecraftServer.sleepForTick(MinecraftServer.java:1354)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1265)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at java.base/java.lang.Thread.run(Thread.java:831)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] Caused by: java.lang.reflect.InvocationTargetException
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:78)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at ru.armagidon.poseplugin.api.utils.nms.ToolFactory.create(ToolFactory.java:71)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at ru.armagidon.poseplugin.api.poses.seatrequiring.LayPose.<init>(LayPose.java:23)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] ... 30 more
[Mon, 12. Jul 2021 12:59:33 UTC WARN] Caused by: java.lang.NoSuchMethodError: 'void net.minecraft.network.protocol.game.PacketPlayOutEntityDestroy.<init>(int)'
[Mon, 12. Jul 2021 12:59:33 UTC WARN] at ru.armagidon.poseplugin.api.utils.nms.v1_17.npc.FakePlayer117.<init>(FakePlayer117.java:122)
[Mon, 12. Jul 2021 12:59:33 UTC WARN] ... 37 more
```

The following poses also do not work: `/handshake`, `/clap`, `/point`, `/spinjitsu`, `/wave`.
Causes errors similar to `/lay`.
logs
```
[01:30:43 INFO]: Disguys issued server command: /spinjitsu
[01:30:43 WARN]: java.lang.reflect.InvocationTargetException
[01:30:43 WARN]: at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[01:30:43 WARN]: at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:78)
[01:30:43 WARN]: at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[01:30:43 WARN]: at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
[01:30:43 WARN]: at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
[01:30:43 WARN]: at ru.armagidon.poseplugin.api.poses.PoseBuilder.build(PoseBuilder.java:40)
[01:30:43 WARN]: at ru.armagidon.poseplugin.plugin.commands.PoseCommandGenerator.lambda$generatePoseCommands$4(PoseCommandGenerator.java:63)
[01:30:43 WARN]: at ru.armagidon.poseplugin.plugin.commands.SimpleCommand.lambda$new$1(SimpleCommand.java:39)
[01:30:43 WARN]: at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[01:30:43 WARN]: at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
[01:30:43 WARN]: at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:821)
[01:30:43 WARN]: at org.bukkit.Bukkit.dispatchCommand(Bukkit.java:819)
[01:30:43 WARN]: at org.bukkit.command.FormattedCommandAlias.execute(FormattedCommandAlias.java:37)
[01:30:43 WARN]: at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
[01:30:43 WARN]: at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:821)
[01:30:43 WARN]: at net.minecraft.server.network.PlayerConnection.handleCommand(PlayerConnection.java:2248)
[01:30:43 WARN]: at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2059)
[01:30:43 WARN]: at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2040)
[01:30:43 WARN]: at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:46)
[01:30:43 WARN]: at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:6)
[01:30:43 WARN]: at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$ensureRunningOnSameThread$1(PlayerConnectionUtils.java:56)
[01:30:43 WARN]: at net.minecraft.server.TickTask.run(TickTask.java:18)
[01:30:43 WARN]: at net.minecraft.util.thread.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:149)
[01:30:43 WARN]: at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.executeTask(IAsyncTaskHandlerReentrant.java:23)
[01:30:43 WARN]: at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:1407)
[01:30:43 WARN]: at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:190)
[01:30:43 WARN]: at net.minecraft.util.thread.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:122)
[01:30:43 WARN]: at net.minecraft.server.MinecraftServer.bf(MinecraftServer.java:1385)
[01:30:43 WARN]: at net.minecraft.server.MinecraftServer.executeNext(MinecraftServer.java:1378)
[01:30:43 WARN]: at net.minecraft.util.thread.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:132)
[01:30:43 WARN]: at net.minecraft.server.MinecraftServer.sleepForTick(MinecraftServer.java:1354)
[01:30:43 WARN]: at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1265)
[01:30:43 WARN]: at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317)
[01:30:43 WARN]: at java.base/java.lang.Thread.run(Thread.java:831)
[01:30:43 WARN]: Caused by: java.lang.reflect.InvocationTargetException
[01:30:43 WARN]: at jdk.internal.reflect.GeneratedConstructorAccessor50.newInstance(Unknown Source)
[01:30:43 WARN]: at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[01:30:43 WARN]: at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
[01:30:43 WARN]: at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
[01:30:43 WARN]: at ru.armagidon.poseplugin.api.utils.nms.ToolFactory.create(ToolFactory.java:71)
[01:30:43 WARN]: at ru.armagidon.poseplugin.api.poses.experimental.SpinJitsuPose.<init>(SpinJitsuPose.java:36)
[01:30:43 WARN]: ... 34 more
[01:30:43 WARN]: Caused by: java.lang.NoSuchMethodError: 'void net.minecraft.network.protocol.game.PacketPlayOutEntityDestroy.<init>(int)'
[01:30:43 WARN]: at ru.armagidon.poseplugin.api.utils.nms.v1_17.npc.FakePlayer117.<init>(FakePlayer117.java:122)
[01:30:43 WARN]: ... 40 more
```

--- a/poseplugin-1_17/pom.xml
+++ b/poseplugin-1_17/pom.xml
@@ -16,11 +16,84 @@
         <maven.compiler.target>16</maven.compiler.target>
     </properties>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <release>16</release>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.3.0-SNAPSHOT</version>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <createDependencyReducedPom>false</createDependencyReducedPom>
+
+                            <artifactSet>
+                                <excludes>
+                                    <exclude>junit:junit</exclude>
+                                    <exclude>org.hamcrest:hamcrest-core</exclude>
+                                    <exclude>com.googlecode.json-simple:json-simple</exclude>
+                                </excludes>
+                            </artifactSet>
+
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>net.md-5</groupId>
+                <artifactId>specialsource-maven-plugin</artifactId>
+                <version>1.2.2</version>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>remap</goal>
+                        </goals>
+                        <id>remap-obf</id>
+                        <configuration>
+                            <srgIn>org.spigotmc:minecraft-server:1.17.1-R0.1-SNAPSHOT:txt:maps-mojang</srgIn>
+                            <reverse>true</reverse>
+                            <remappedDependencies>org.spigotmc:spigot:1.17.1-R0.1-SNAPSHOT:jar:remapped-mojang</remappedDependencies>
+                            <remappedArtifactAttached>true</remappedArtifactAttached>
+                            <remappedClassifierName>remapped-obf</remappedClassifierName>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>remap</goal>
+                        </goals>
+                        <id>remap-spigot</id>
+                        <configuration>
+                            <inputFile>${project.build.directory}/${project.artifactId}-${project.version}-remapped-obf.jar</inputFile>
+                            <srgIn>org.spigotmc:minecraft-server:1.17.1-R0.1-SNAPSHOT:csrg:maps-spigot</srgIn>
+                            <remappedDependencies>org.spigotmc:spigot:1.17.1-R0.1-SNAPSHOT:jar:remapped-obf</remappedDependencies>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
     <dependencies>
         <dependency>
             <groupId>org.spigotmc</groupId>
             <artifactId>spigot</artifactId>
-            <version>1.17-R0.1-SNAPSHOT</version>
+            <version>1.17.1-R0.1-SNAPSHOT</version>
             <classifier>remapped-mojang</classifier>
             <scope>provided</scope>
         </dependency>

--- a/poseplugin-1_17/src/main/java/ru/armagidon/poseplugin/api/poses/crawl/v1_17/CrawlHander117.java
+++ b/poseplugin-1_17/src/main/java/ru/armagidon/poseplugin/api/poses/crawl/v1_17/CrawlHander117.java
@@ -4,7 +4,7 @@ package ru.armagidon.poseplugin.api.poses.crawl.v1_17;
 import lombok.SneakyThrows;
 import net.minecraft.core.Direction;
 import net.minecraft.network.protocol.game.ClientboundAddMobPacket;
-import net.minecraft.network.protocol.game.ClientboundRemoveEntityPacket;
+import net.minecraft.network.protocol.game.ClientboundRemoveEntitiesPacket;
 import net.minecraft.network.protocol.game.ClientboundSetEntityDataPacket;
 import net.minecraft.network.protocol.game.ClientboundTeleportEntityPacket;
 import net.minecraft.server.level.ServerPlayer;
@@ -102,7 +102,7 @@ public class CrawlHander117 extends CrawlHandler
 
         @Override
         public void hide0() {
-            ClientboundRemoveEntityPacket destroy = new ClientboundRemoveEntityPacket(shulker.getId());
+            ClientboundRemoveEntitiesPacket destroy = new ClientboundRemoveEntitiesPacket(shulker.getId());
             FakePlayer117.sendPacket(player, destroy);
         }
 

--- a/poseplugin-1_17/src/main/java/ru/armagidon/poseplugin/api/utils/nms/v1_17/npc/FakePlayer117.java
+++ b/poseplugin-1_17/src/main/java/ru/armagidon/poseplugin/api/utils/nms/v1_17/npc/FakePlayer117.java
@@ -76,7 +76,7 @@ public class FakePlayer117 extends FakePlayer<SynchedEntityData>
     private final ClientboundPlayerInfoPacket removeNPCData;
     private ClientboundAddPlayerPacket spawner;
     private ClientboundMoveEntityPacket.PosRot movePacket;
-    private final ClientboundRemoveEntityPacket destroy;
+    private final ClientboundRemoveEntitiesPacket destroy;
 
     public FakePlayer117(Player parent, Pose pose) {
         super(parent, pose);
@@ -119,7 +119,7 @@ public class FakePlayer117 extends FakePlayer<SynchedEntityData>
 
         this.inventory = new NPCInventory117(this);
 
-        this.destroy = new ClientboundRemoveEntityPacket(fake.getId());
+        this.destroy = new ClientboundRemoveEntitiesPacket(fake.getId());
 
         //Create instance of move packet to pop up npc a little
         this.movePacket = new ClientboundMoveEntityPacket.PosRot(fake.getId(), (short) 0,(short)2,(short)0,(byte)0,(byte)0, true);

--- a/poseplugin/pom.xml
+++ b/poseplugin/pom.xml
@@ -115,9 +115,9 @@
                         </goals>
                         <id>remap-obf</id>
                         <configuration>
-                            <srgIn>org.spigotmc:minecraft-server:1.17-R0.1-SNAPSHOT:txt:maps-mojang</srgIn>
+                            <srgIn>org.spigotmc:minecraft-server:1.17.1-R0.1-SNAPSHOT:txt:maps-mojang</srgIn>
                             <reverse>true</reverse>
-                            <remappedDependencies>org.spigotmc:spigot:1.17-R0.1-SNAPSHOT:jar:remapped-mojang</remappedDependencies>
+                            <remappedDependencies>org.spigotmc:spigot:1.17.1-R0.1-SNAPSHOT:jar:remapped-mojang</remappedDependencies>
                             <remappedArtifactAttached>true</remappedArtifactAttached>
                             <remappedClassifierName>remapped-obf</remappedClassifierName>
                         </configuration>
@@ -130,8 +130,8 @@
                         <id>remap-spigot</id>
                         <configuration>
                             <inputFile>${project.build.directory}/${project.artifactId}-${project.version}-remapped-obf.jar</inputFile>
-                            <srgIn>org.spigotmc:minecraft-server:1.17-R0.1-SNAPSHOT:csrg:maps-spigot</srgIn>
-                            <remappedDependencies>org.spigotmc:spigot:1.17-R0.1-SNAPSHOT:jar:remapped-obf</remappedDependencies>
+                            <srgIn>org.spigotmc:minecraft-server:1.17.1-R0.1-SNAPSHOT:csrg:maps-spigot</srgIn>
+                            <remappedDependencies>org.spigotmc:spigot:1.17.1-R0.1-SNAPSHOT:jar:remapped-obf</remappedDependencies>
                         </configuration>
                     </execution>
                 </executions>

